### ISSUE TITLE
Always show scrollbar / scrollbar width

### ISSUE THREAD

**Issue by ceonelson** [axcore/tartube#401]
Would it be possible to have an option to always show the scrollbars and possibly set their width? There's times I am on a bad connection and trying to get the mouse to line up on the few pixel border to get it to show the scrollbar, then grab the scroll handle to scroll where I need to is quite difficult!

Not to mention on some of the settings screens things line up perfectly where I didn't realize there were more options to scroll to until I accidentally rolled the mouse wheel...

Thanks Axcore!

**Comment by axcore** [comment]
No, it's not possible. The geniuses who maintain the Gtk graphics libraries think that a disappearing scrollbar is a brilliant idea, so there is no way to change it.

You can set an environment variable, or you can override the Gtk theme. Both are described [here](https://ubuntuhandbook.org/index.php/2019/09/make-scrollbar-always-visible-ubuntu-18-04/).

**Comment by ceonelson** [comment]
Yes I hate disappearing scrollbars too, I'm glad that environment variable works as that makes it so much more usable, thanks Axcore!
